media-centre/speeches/_posts/2022-10-08-post-Opening Remarks by Minister Indranee Rajah at Celebrating Our Grands: A Song From Me To You 2022.md

```diff
@@ -0,0 +1,32 @@
+Mr Kho Choon Keng, President of the Singapore Chinese Chamber of Commerce & Industry (SCCCI)<br>
+Distinguished Guests<br>
+Ladies and Gentlemen<br>
+
+It is a wonderful afternoon to be here with you all. When you are here at SCCCI, you really feel the weight of history. The Chamber was founded in 1906 and it has such a long history in association with Singapore. It is also closely linked to many of our families in Singapore.
+
+When I was giving out the Pioneer Generation card during SG50 celebrations,  one of the gentlemen I gave the card to was actually the great great-grandson of Mr Kim Tian. That is, Kim Tian of Kim Tian road. The Kim Tian family was still living near and around Kim Tian road. The original Kim Tian was a ship chandler, so he had been doing business here in Singapore. 
+
+Another family associated with the Tiong Bahru area that I look after, is the Tan Tock Seng family; not Tan Tock Seng himself, but his grandsons who were businessmen. Around that same time, the entire Tan clan had come back to Singapore together at the Tiong Bahru area. All the businessmen and businesses that came to Singapore in those early years, they settled here. They set up families; their families are part and parcel of our history and our nation. They are not only part of our past, but part of our future. 
+
+And that is why today's event is so meaningful, with a song from me to you to grandparents, because if you think about it, our grandparents would have arrived at a point in life where they have done what they need to do for their careers. They are at a time when they can sit back, relax, enjoy things, hopefully a little bit more. And actually, the thing that they most look forward to, is time with their children and their grandchildren. If you look at it from their perspective, we, who are their children and their grandchildren, are actually their greatest legacy to Singapore, which is why this event is very meaningful, and I'm very glad to be part of it. 
+
+It also brings me to the topic of families, because here in Singapore, we see families as part of our social structure. It is families that anchor us, it is families that give us our sense of belonging. Families are the ones that we work for. But why do you go to work and work so hard? Why do you do what you do? It is not only for yourself, it is for your families - whether it is for your parents, your wife, your children or your husband. It is for the people who we feel that we belong to. And for that reason, we want to make sure that families are well supported, and that Singapore can be a place where families can grow and flourish. Not to mention, of course, that our total fertility rate could do with some improving. And that is why the Ministry of Family and Social Development has designated this year as the Year of Celebrating Families, to let families know how important they are, and hopefully to signal to families to appreciate one another.
```

Comment:
Ministry of Social and Family Development - (check this. In text it was listed as Ministry of Family and Social Development)